### PR TITLE
docs(multiple_testing_power): update documentations

### PR DESCRIPTION
**Pull request by xyzou685**
To help us get this pull request reviewed and merged quickly, please be sure to include the following items:

* [ ] Tests (if applicable)
* [x] Documentation (if applicable)
* [ ] A full explanation here in the PR description of the work done and statistical methods used

## PR Type
What kind of change does this PR introduce?

* [ ] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [x] Documentation content changes
* [ ] Tests
* [ ] Other

## Backward Compatibility

Is this change backward compatible with the most recently released version? Does it introduce changes which might change the user experience in any way? Does it alter the API in any way?

* [ ] Yes (backward compatible)
* [ ] No (breaking changes)

### CODE REVIEW

**Comment by xyzou685** [comment]
We will need to determine how many metrics and variants or metrics*(variants-1) we can support at most. After that I will add these constraints to ```input_utils.py```

File: sample_size/metrics_schema.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                            ]
+                        }
+                    }
+                }
+            }
+        ],
+        "required": ["metric_type", "metric_metadata"]
+    },
+    "minItems": 1
+}
```
Comment (dileeppandey) [inline sample_size/metrics_schema.json:84]:
```suggestion
}

```

Comment (dileeppandey) [reply sample_size/metrics_schema.json:84]: good idea to have an eol in the file

File: Simulation Validation/Validation of test statistics simulation.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+```python
+import numpy as np
+from scipy.stats import nct, ks_2samp
+from itertools import product
+from tqdm import tqdm
+import pandas as pd
+import seaborn as sns
+
+#sampling from observations
+def obs_sample(δ, σ, n, size):
```
[the remark this answers is not in the document]
Comment (xyzou685) [reply Simulation Validation/Validation of test statistics simulation.md:36]: Using names will have long tick labels on graphs and might make it hard to read. Does it look better after I added the docstring?

**Review by vjohnson1-godaddy — APPROVED** [review]

**Review by dfu-godaddy — APPROVED** [review]

File: Simulation Validation/Validation of test statistics simulation.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+import seaborn as sns
+
+
+def obs_sample(δ, σ, n, size):
+    """
+    This function samples from normally distributed numeric metric's observations under true alternative hypothesis and calculates the samples' p-values
+
+    Attributes:
+    δ: the mean's shift from null hypothesis
+    σ: standard deviation
```
Comment (dileeppandey) [inline Simulation Validation/Validation of test statistics simulation.md:42]:
```suggestion
    δ: delta, the mean's shift from null hypothesis
    σ: sigma, standard deviation
```
?

Comment (dileeppandey) [reply Simulation Validation/Validation of test statistics simulation.md:42]: is there a way to alias on the plots though? i am okay if we decide not to do this

**Review by dileeppandey — APPROVED** [review]
with a nit comment, if we address it, good, otherwise the pr lgtm

**Pull request merged**